glossary/ja/terms/citizen-engagement/index.md

```diff
@@ -0,0 +1,7 @@
+---
+section: terms
+lang: ja
+title: Citizen Engagement 
+---
+
+市民参画。市民を政策形成や市政の意思決定の場面に積極的に参加させる取り組み。市民参画は、[open government](/glossary/en/terms/open-government/)が目指す中心的な目的である。政策における意思決定をよりよいものに変える、市民の賛意と支持を獲得・維持する、という目的と並ぶものである。オープンデータは、情報に基づいた市民参画を促す上で重要なツールである。
```

Comment:
Link of Open government is to the term in English, do you want to change it to Japanese?